dashboard/serializers.py

```diff
@@ -10,70 +10,66 @@ class UserProgramSearchSerializer:
     """
     Provides functions for serializing a ProgramEnrollment for the ES index
     """
+    PAID_STATUS = 'Paid'
+    UNPAID_STATUS = 'Auditing'
+
     @classmethod
     def serialize_enrollments(cls, mmtrack):
         """
-        Serializes a user's enrollment data for search results
+        Serializes a user's enrollment data for search results in such a way that enrollments
+        in multiple runs of a single course will result in just one serialization.
 
         Args:
             mmtrack (MMTrack): An MMTrack object
         Returns:
-            list: Serialized courses
+            list: Serialized course enrollments
         """
-        enrollment_status_map = {}
-        course_runs = mmtrack.get_all_enrolled_course_runs()
-        for course_run in course_runs:
+        serialized_enrollments_map = {}
+        for course_run in mmtrack.get_all_enrolled_course_runs():
             course_title = course_run.course.title
-            is_verified = mmtrack.is_enrolled_mmtrack(course_run.edx_course_key)
-            # If any course run for this course was verified/paid, maintain the verified status
-            enrollment_status_map[course_title] = enrollment_status_map.get(course_title) or is_verified
-        serialized_enrollments = []
-        for course_title, is_verified in enrollment_status_map.items():
-            enrollment_status = 'Paid' if is_verified else 'Auditing'
-            serialized_enrollments.extend([
-                {'level': 1, 'value': course_title, 'ancestors': []},
-                {'level': 2, 'value': enrollment_status, 'ancestors': [course_title]}
-            ])
-        return serialized_enrollments
+            # If any course run for this course was verified/paid, count it as verified
+            if course_title not in serialized_enrollments_map or \
```

Comment:
if anything we should use `course_run.course.id`. we only want a single enrollment item for multiple runs of the same course. i'll make this change